Flow.Launcher.Infrastructure/Storage/JsonStorage.cs

```diff
@@ -24,10 +24,9 @@ internal JsonStrorage()
         {
             // use property initialization instead of DefaultValueAttribute
             // easier and flexible for default value of object
-            _serializerSettings = new JsonSerializerSettings
+            _serializerSettings = new JsonSerializerOptions
             {
-                ObjectCreationHandling = ObjectCreationHandling.Replace,
-                NullValueHandling = NullValueHandling.Ignore
+                IgnoreNullValues = false
```

Comment:
I am assuming these options and comment are no longer applicable?

Reply:
yes, the default behavior of System.Text.Json is replace, and ignore doesn't work well as my comment before.